src/compute/src/render/reduce.rs

```diff
@@ -1460,6 +1511,35 @@ where
     }
 }
 
+/// Evaluates the fused MFP, if one exists, on a reconstructed `DatumVecBorrow`
+/// containing key and aggregate values, then returns a result `Row` or `None`
+/// if the MFP filters the result out.
+fn evaluate_mfp_after<'a, 'b>(
+    mfp_after: &'a Option<SafeMfpPlan>,
+    datums_local: &'b mut mz_repr::DatumVecBorrow<'a>,
+    temp_storage: &'a RowArena,
+    key_len: usize,
+) -> Option<Row> {
+    let binding = SharedRow::get();
+    let mut row_builder = binding.borrow_mut();
+    let mut row_packer = row_builder.packer();
+    // Apply MFP if it exists and pack a Row of
+    // aggregate values from `datums_local`.
+    if let Some(mfp) = mfp_after {
+        // It is fine to ignore errors here, as they are scanned
+        // for elsewhere if the MFP can error.
```

Comment:
Not only fine to ignore errors, but required I think? The comment makes it sound we chose not to, but we simply can't.

Reply:
Yup, changed the language to make that clear.